src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.spring.dto.request;
 
+import java.util.Objects;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 4, max = 255)
     private String password;
+    @Size(min = 8, max = 255)
```

Comment:
you have validation for the length of password in the validator, remove it from here